ClosedXML/Excel/CalcEngine/Functions/Financial.cs

```diff
@@ -60,5 +64,31 @@ public static void Register(CalcEngine ce)
             // YIELDDISC Returns the annual yield for a discounted security; for example, a Treasury bill
             // YIELDMAT Returns the annual yield of a security that pays interest at maturity
         }
+
+        private static object Payment(List<Expression> p)
+        {
+            double rate = 1.0 + p[0];
```

Comment:
I'd prefer `double rate = p[0];` with adjustments below accordingly.

Reply:
And then when `rate == 0.0` none of the `Math.Pow()` functions would be needed.

Reply:
Haven't noticed it. Thanks.